openspec/changes/project-02-plan-root-command-fix/specs/project-command-surface/spec.md

```
@@ -0,0 +1,8 @@
+## ADDED Requirements
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)